HISTORY.md

```diff
@@ -1,6 +1,27 @@
 ---
 title: Release History
 ---
+# 3.0.0a1 (2024-08-01)
+## Highlights
+Mesa 3.0 alpha 1 (`v3.0.0a1`) is another step towards our next major version. This release introduces a name change from JupyterViz (jupyter_viz) to SolaraViz (solara_viz), to better represent the tech stack being used. It also includes two bugfixes also present in 2.3.2.
+
+## What's Changed
+### ⚠️ Breaking changes
+* viz: Combine code for rendering in browser and Jupyter by @rht in https://github.com/projectmesa/mesa/pull/2180
+* refactor: Rename jupyter_viz namespace to solara_viz by @rht in https://github.com/projectmesa/mesa/pull/2188
```

Comment:
I wouldn't consider this a breaking change. I don't think people ever use `from mesa.visualization.jupyter_viz import ...`. It's never advertised in the tutorial nor API documentation.

Reply:
Moved it